masonry_core/src/testing/harness.rs

```diff
@@ -139,24 +138,18 @@ pub struct TestHarnessParams {
 /// Assert a snapshot of a rendered frame of your app.
 ///
 /// This macro takes a test harness and a name, renders the current state of the app,
-/// and stores the render as a PNG next to the test, in a `./screenshots/` folder.
+/// and stores the render as a PNG next to the test, in a `<CRATE_ROOT>/screenshots/` folder.
```

Comment:
I don't think "next to" here is right.
```suggestion
/// and compares against the PNG file with the name in the folder `<CRATE_ROOT>/screenshots/` folder.
```